ldc-config/ldc2.conf

```diff
@@ -4,16 +4,21 @@
 // The default group is required
 default:
 {
-    // 'switches' holds array of string that are appends to the command line
-    // arguments before they are parsed.
+    // default switches injected before all explicit command-line switches
     switches = [
+        "-defaultlib=phobos2-ldc,druntime-ldc",
+    ];
+
+    // default switches appended after all explicit command-line switches
+    post-switches = [
         "-I%%ldcbinarypath%%/../include/d/ldc",
         "-I%%ldcbinarypath%%/../include/d",
         "-L-L%%ldcbinarypath%%/../lib",
         "-L-L%%ldcbinarypath%%/../lib32",
         "-L-L%%ldcbinarypath%%/../lib64",
         "-L--no-warn-search-mismatch",
-        "-defaultlib=phobos2-ldc,druntime-ldc",
-        "-debuglib=phobos2-ldc-debug,druntime-ldc-debug"
     ];
+
+    // default rpath when linking against the shared default libs
+    rpath = "%%ldcbinarypath%%/../lib:%%ldcbinarypath%%/../lib32:%%ldcbinarypath%%/../lib64";
```

Comment:
Note, the extra `lib64` is there to support i386 packages' multilib builds.

Reply:
Looks fine. FWIW, you can use `-DLIB_SUFFIX={32,64}` in the CMake command line to deploy the native libs to `lib<LIB_SUFFIX>`, so that you end up consistently with `lib32` and `lib64` across architectures.